tests/fixtures/compensated_events.py

```diff
@@ -0,0 +1,35 @@
+from cellengine.resources.fcs_file import FcsFile
+import json
+from cellengine.utils.parse_fcs_file import parse_fcs_file
+from cellengine.resources.compensation import Compensation
+import pytest
+from pandas import read_csv, DataFrame
+
+
+@pytest.fixture(scope="function")
+def acea_events() -> DataFrame:
+    """Real events from 'Acea - Novocyte.fcs'"""
+    events_body = open("tests/data/Acea - Novocyte.fcs", "rb")
+    return parse_fcs_file(events_body.read())
+
+
+@pytest.fixture(scope="function")
+def acea_fcs_file(acea_events):
+    with open("tests/data/acea.json", "r") as f:
+        fcs_file = json.load(f)
+    file = FcsFile.from_dict(fcs_file)
+    file.events = acea_events
+    return file
+
+
+@pytest.fixture(scope="function")
+def acea_compensation(acea_fcs_file):
+    return Compensation.from_spill_string(acea_fcs_file.spill_string)
+
+
+@pytest.fixture(scope="function")
+def acea_events_compensated(acea_events):
+    """File-internal compensated events from 'Acea - Novocyte.fcs'"""
+    file = read_csv("tests/data/Acea_compensated.zip", sep="\t")
```

Comment:
Not a priority right now, but if anyone complains about package size, it's a waste to check in a 30 MB file to only use the first row. Could just use `head -n 5` on the file or hard-code the values into a .py file here.